faunadb/query.go

```diff
@@ -1238,3 +1238,48 @@ func Select(path, value interface{}, options ...OptionalParameter) Expr {
 func SelectAll(path, value interface{}) Expr {
 	return fn2("select_all", path, "from", value)
 }
+
+// ToString attempts to convert an expression to a string literal.
```

Comment:
This docs are a little confusing to me. It think it's because of the "expression" word. What gets converted is the resulting value right?

Maybe something on the lines of "Attempts to convert a value to a string literal" would be more user friendly? Idk.

Reply:
They accept any expression at all - refs, collections, queries, scalars - anything. Each of the conversion functions fails when the provided expression cannot be converted to the appropriate type. The rules for which types can be converted to strings, for instance, may be loosened in the future.

Reply:
I guess my point is that "expression" can be confusing to a database user. It's more of a programming language terminology rather than a database one. If we just say `ToString` attempts to convert a value to a string, it is easier to understand. Like "hey, I have this number in the database, now I can convert it to a string". Even if it doesn't convey all the semantics of the `ToString` function itself.

But I might be nitpicking here. Everything else looks good. Feel free to merge.